content/workshops/intro-to-linux/_index.md

```diff
@@ -1,12 +1,14 @@
 ---
 title: Introduction to Linux
+ready: true
 ---
 
-- presentation: https://docs.google.com/presentation/d/1dhj1-p9Ufl4ilSy_Nu8sw-kuNVWkSvhp0dn5KGo9TyY
-- post install guide 1: https://gitlab.com/prelude-open-source/mint-post-install-guide
-- post install guide 2: https://fossbytes.com/things-to-do-after-installing-linux-mint/
-
-
+ 1. [Introduction](https://docs.google.com/presentation/d/1dhj1-p9Ufl4ilSy_Nu8sw-kuNVWkSvhp0dn5KGo9TyY)
+ 2. [post install guide 1](https://gitlab.com/prelude-open-source/mint-post-install-guide)
```

Comment:
The numbers in this comment relate to the numbers in this file.

1. Embed this as an iframe. Because it is easy and it looks good. Open the presentation, click file. click "publish to the web". click "embed". select the medium size
2. This content is more a TOPIC than a WORKSHOP. If you are consistent then everyone will know how to find what they need
3. This content is TOPIC content. Are you going to read this to the recruits? 
4. iframe. Also you are giving people edit rights to the presentation.
5. see 4
6. see 4

It would also be god to make this stuff more searchable. Either:
a) make a directory for linux worekshops and then include each workshop as a seperate markdown file inside the directory
b) give each presentation a heading

In either case: 
- run `hugo serve` and make sure it looks good
- try using the search bar in the side panel to see how easy it is to find your presentations

Reply:
Also, please pull master just in case